code/modules/mob/living/death.dm

```diff
@@ -69,6 +69,8 @@
  * * gibbed - Was the mob gibbed?
 */
 /mob/living/proc/death(gibbed)
+	SEND_SIGNAL(src, COMSIG_LIVING_PREDEATH, gibbed)
```

Comment:
@ninjanomnom I wanna know if you'd rather see this handled some other way. See the commit message of https://github.com/tgstation/tgstation/pull/55903/commits/ba3099b1f8b6e4bd15b2e5629a3e1b941213ab6d

Reply:
Do you want to use the `gibbed` value? Else `set_stat` sends a signal already.

Reply:
If that can be done I think it would be preferable. This seems heavy-handed.